thinc/layers/with_flatten.py

```diff
@@ -2,49 +2,64 @@
 
 from ..model import Model
 from ..config import registry
-from ..types import ListXd
 
 
+InItemT = TypeVar("InItemT")
+OutItemT = TypeVar("OutItemT")
 ItemT = TypeVar("ItemT")
-InT = Sequence[Sequence[ItemT]]
-OutT = TypeVar("OutT", bound=ListXd)
+
+NestedT = List[List[ItemT]]
```

Comment:
Missed this one:
```suggestion
NestedT = Sequence[Sequence[ItemT]]
```

Reply:
Right, I used to have `NestedT` and `NestedSeqT` aliases, but I simplified it and forgot to update the docs.

Reply:
Ok, fixed in the docs.